### ISSUE TITLE
about PSFHS

### ISSUE THREAD

**Issue by Saillxl** [maskoffs/PS-FH-MICCAI23#1]
Thank you for your contribution. May I ask how to conduct data pre-processing for PSFHS? Or how to convert .mha data into photos?

**Comment by maskoffs** [comment]
Sure! first, you can install SimpleITK package using the command "pip install simpleitk". Then, you can use "SimpleITK.ReadImage()" function to read files of .mha format，and use the "SimpleITK.GetArrayFromImage()" to transfer image to numpy array.

**Comment by Saillxl** [comment]
Thank you for your prompt reply!  After processing with SimpleITK.GetArrayFromImage(), can I convert the numpy array to PNG images for storage and then used with training?

**Comment by maskoffs** [comment]
yes, we usually use opencv package to save numpy array as image of png format.

**Comment by Saillxl** [comment]
Thanks for your patience! I got it.
